cypress/integration/tests/manual-lice-register.spec.js

```diff
@@ -0,0 +1,80 @@
+import {LoginPage} from "../../support/pageobjects/login-page";
+import {ManualLiceRegister} from "../../support/pageobjects/manual-lice-register";
+
+const locationId = 2127;
```

Comment:
hardcoded id is not good at all. 
Why don't you set it up? or if it's something global like login, move it to config file?